.github/workflows/tests-rdbstorage.yml

```diff
@@ -0,0 +1,103 @@
+name: Tests (RDB storage)
+
+on:
+  push:
+    branches:
+      - master
+  pull_request: {}
+  schedule:
+    - cron: '0 23 * * SUN-THU'
+
+jobs:
+  tests-rdbstorage:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    services:
+      mysql:
+        image: mysql:5.7
+        ports:
+          - 3306:3306
+        env:
+          MYSQL_ROOT_PASSWORD: mandatory_arguments
+          MYSQL_DATABASE: optunatest
+          MYSQL_USER: user
+          MYSQL_PASSWORD: test
+        options: >-
+          --health-cmd "mysqladmin ping -h localhost"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
```

Comment:
I will add why I added this health check.

Reply:
Most examples use health checks. https://freek.dev/1590-how-to-use-a-mysql-database-on-github-actions
The benefit is that we can find an error at the "Initialize containers" step if containers are not running correctly.